index.js

```diff
@@ -17,7 +17,10 @@ const async = require('async');
 const zlib = require('zlib');
 
 const m_alServiceC = require('al-collector-js/al_servicec');
+const m_alAzCollectC = require('al-collector-js/azcollectc');
```

Comment:
the two vars above aren't used anywhere.

Reply:
Those and more apprently. deleting those suddenly made the linter wake up and show a few more...